grandnode/grandnode
# #481 Product Size Chart

Resolves #481 
Type: feature

## Issue
Allow upload and display of an image-based size chart.

## Solution
Added size chart to admin area and simple and grouped product views

## Breaking changes
None

## Testing
1. Create a simple product
2. Add a Size Chart picture under Additional Info
3. View on storefront, confirm size chart is available below product image
4. Repeat 1-3 above for grouped product
5. Copy product
6. Ensure that size chart image is copied in DB and copy is available in new product
